src/rate_limiter/src/lib.rs

```diff
@@ -560,6 +567,23 @@ pub(crate) mod tests {
         }
     }
 
+    #[test]
+    fn test_token_bucket_auto_replenish() {
+        const SIZE: u64 = 1000;
+        const TIME: u64 = 1000;
```

Comment:
With these values we get 1 new token every millisecond.

with:

```suggestion
        const SIZE: u64 = 1;
        const TIME: u64 = 1000;
```

we should have 1 new token every second, so it's easier to trigger the corner-case condition.

Reply:
I understand your point, but `autoreplenish()` takes `~45ns` so a `1ms` step is distant enough (`x22222`) from this while still a reasonable use case.

Reply:
I think this is resolved by using your suggestion in https://github.com/firecracker-microvm/firecracker/pull/3370#discussion_r1072304501.

Reply:
When I changed the value to the suggested ones the test was actually failing:

```
running 1 test
test tests::test_token_bucket_auto_replenish ... FAILED

failures:

---- tests::test_token_bucket_auto_replenish stdout ----
thread 'tests::test_token_bucket_auto_replenish' panicked at 'assertion failed: `(left == right)`
  left: `1`,
 right: `0`', src/rate_limiter/src/lib.rs:584:9
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
```

Reply:
This relates to https://github.com/firecracker-microvm/firecracker/pull/3370#discussion_r1072332075